### PR TITLE
@craigspaeth => Auction2 "My active bids"

### PR DESCRIPTION
**Pull request by sweir27**
This PR adds the "My Active Bids" section to the auction2 page, based on the new designs.
image

### CODE REVIEW

File: assets/auction2.js [hunk 1]
```diff
@@ -1,2 +1 @@
-// TODO: Add Babel back in assets pipeline
```
Comment (sweir27) [inline assets/auction2.js]: 🙌 w00t!

File: apps/auction2/client.js [hunk 2]
```diff
@@ -1,8 +1,24 @@
 import Auction from '../../models/auction.coffee'
+import CurrentUser from '../../models/current_user.coffee'
 import ClockView from '../../components/clock/view.coffee'
 import { data as sd } from 'sharify'
 import _ from 'underscore'
+import MyActiveBids from '../../components/my_active_bids/view.coffee'
+
+const myActiveBidsTemplate = require('./templates/my_active_bids.jade')
```
Comment (craigspaeth) [inline apps/auction2/client.js]: 🤔  I assume this is b/c Browserify was having trouble parsing `import`s for jade files? I wonder if we just need to swap the order of the transform.s

File: apps/auction2/test/my_active_bids.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import Auction from '../../../models/auction.coffee'
+import CurrentUser from '../../../models/current_user.coffee'
+import { resolve } from 'path'
+import MyActiveBidsView from '../../../components/my_active_bids/view.coffee'
+import Backbone from 'backbone'
+import { data as sd } from 'sharify'
+
+const template = require('jade').compileFile(require.resolve('../templates/my_active_bids.jade'))
+
+describe('my active bids auction page template', () => {
```
Comment (craigspaeth) [inline apps/auction2/test/my_active_bids.js]: 👌 nice set of tests!

File: lib/setup.coffee [hunk 4]
```diff
@@ -114,7 +114,7 @@ module.exports = (app) ->
       dest: path.resolve(__dirname, "../public")
     app.use require("browserify-dev-middleware")
       src: path.resolve(__dirname, "../")
-      transforms: [require("jadeify"), require('caching-coffeeify')]
+      transforms: [require("jadeify"), require('caching-coffeeify'), require('babelify')]
```
Comment (craigspaeth) [inline lib/setup.coffee]: Moving jadeify to the end might work around the `require` thing I mentioned above.

**Comment by craigspaeth** [comment]
👏  looks great! Comments aren't blockers, so feel free to address what you like and merge when you're ready. Solid!

**Comment by sweir27** [comment]
Even moving `jadeify` didn't let me change that require line, so I'm merging for now but let me know if you want to look at it together!

**Pull request merged**